src/pages/myGeotab/addIns/mapAddIns.tsx

```diff
@@ -433,11 +444,12 @@ const mapAddinServices: ReactNode = (
     <div className="mapAddins__docs-service">
       <div className="mapAddins__docs-service__part">
         <h2>
-          Api Service (
+          Api service (
```

Comment:
```suggestion
          API service (
```
Shouldn't this be all caps since it's an acronym?